data.sql

```diff
@@ -7,3 +7,30 @@ VALUES
     ('Pikachu', '2021-01-07', 1, false, 15.04),
     ('Devimon', '2017-05-12', 5, true, 11);
 
+-- Insert Charmander
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
wow, i never knew this, thank you HAMMAS.